cmd/in/main.go

```diff
@@ -65,7 +65,7 @@ func main() {
 		log.Fatalf("failed to find stemcell matching version: '%s'\n", inRequest.Version.Version)
 	}
 
-	dataLocations := []string{"version", "sha1", "url"}
+	dataLocations := []string{"version", "sha1", "sha256", "url"}
```

Comment:
Do you want to update the README to mention that there's a `sha256` file for `in`/get steps?

Reply:
i already mentioned it in the readme
https://github.com/concourse/bosh-io-stemcell-resource/pull/27/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R42

Reply:
ah derp, my bad! I had hit the "view" checkbox and by the time I got to the last file here I completely forgot about that file! We're all good then!